build/params_wallaby.go

```diff
@@ -0,0 +1,124 @@
+//go:build wallabynet
+// +build wallabynet
+
+package build
+
+import (
+	"os"
+	"strconv"
+
+	"github.com/ipfs/go-cid"
+
+	"github.com/filecoin-project/go-address"
+	"github.com/filecoin-project/go-state-types/abi"
+	"github.com/filecoin-project/go-state-types/network"
+	builtin2 "github.com/filecoin-project/specs-actors/v2/actors/builtin"
+
+	"github.com/filecoin-project/lotus/chain/actors"
+	"github.com/filecoin-project/lotus/chain/actors/policy"
+)
+
+var NetworkBundle = "wallaby"
+var BundleOverrides map[actors.Version]string
+
+const BootstrappersFile = "wallabynet.pi"
+const GenesisFile = "wallabynet.car"
+
+const GenesisNetworkVersion = network.Version16
+
+var UpgradeBreezeHeight = abi.ChainEpoch(-1)
+
+const BreezeGasTampingDuration = 0
+
+var UpgradeSmokeHeight = abi.ChainEpoch(-1)
+var UpgradeIgnitionHeight = abi.ChainEpoch(-2)
+var UpgradeRefuelHeight = abi.ChainEpoch(-3)
+var UpgradeTapeHeight = abi.ChainEpoch(-4)
+
+var UpgradeAssemblyHeight = abi.ChainEpoch(-5)
+var UpgradeLiftoffHeight = abi.ChainEpoch(-6)
+
+var UpgradeKumquatHeight = abi.ChainEpoch(-7)
+var UpgradeCalicoHeight = abi.ChainEpoch(-9)
+var UpgradePersianHeight = abi.ChainEpoch(-10)
+var UpgradeOrangeHeight = abi.ChainEpoch(-11)
+var UpgradeClausHeight = abi.ChainEpoch(-12)
+
+var UpgradeTrustHeight = abi.ChainEpoch(-13)
+
+var UpgradeNorwegianHeight = abi.ChainEpoch(-14)
+
+var UpgradeTurboHeight = abi.ChainEpoch(-15)
+
+var UpgradeHyperdriveHeight = abi.ChainEpoch(-16)
+var UpgradeChocolateHeight = abi.ChainEpoch(-17)
+var UpgradeOhSnapHeight = abi.ChainEpoch(-18)
+var UpgradeSkyrHeight = abi.ChainEpoch(-19)
+
+var DrandSchedule = map[abi.ChainEpoch]DrandEnum{
+	0: DrandMainnet,
+}
+
+var SupportedProofTypes = []abi.RegisteredSealProof{
+	abi.RegisteredSealProof_StackedDrg2KiBV1,
+	abi.RegisteredSealProof_StackedDrg8MiBV1,
+	abi.RegisteredSealProof_StackedDrg512MiBV1,
```

Comment:
we need to support 32 and 64 gib sectors as well, and no 2kib or 8 mib

Reply:
these aren't needed to be specified here, so just gonna remove them